src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1514,7 +1513,7 @@ def convert_av_set_to_managed_disk(resource_group_name, availability_set_name):
 
 # pylint: disable=too-many-locals, unused-argument, too-many-statements
 def create_vm(vm_name, resource_group_name, image=None, size='Standard_DS1_v2', location=None, tags=None, no_wait=False,
-              authentication_type=None, admin_password=None, admin_username=getpass.getuser(), ssh_dest_key_path=None,
```

Comment:
This looks unrelated change. Out of curiosity though, why we remove the default?